docs/getting-started.mdx

```diff
@@ -8,28 +8,59 @@ import ImportantConcepts from '../src/components/getting-started/ImportantConcep
 import Styles from '../src/components/getting-started/Styles';
 
 <Styles />
-# What is Statsig?
-Statsig is the single platform to ship, test and analyze new features. Statsig provides the most advanced [Experimentation](/experiments-plus) and [Feature Flagging](/feature-flags/working-with) tools available, in a platform with full-featured [Product Analytics](/product-analytics/overview), [Session Replay](/session-replay/overview), and more. We believe the best way to build products is with data, and [our mission](https://statsig.com/company) is to democratize data across companies of all sizes. 
+# Welcome to Statsig Documentation
+Statsig is the complete platform for product growth, empowering your team to build and innovate like the best tech companies in the world. 
 
-You can use Statsig with our infrastructure, or plug into your existing metrics with our Warehouse Native product:
+# The Product Growth Kit
+Statsig offers a comprehensive toolkit designed to support every stage of product development and growth:
 
-<ImportantConcepts />
+- **[Feature Flags](/feature-flags/working-with)**: Easily manage feature rollouts, run safe deployments, and control feature access in real-time.
+- **[Experimentation](/experiments-plus)**: Set up and run controlled experiments (A/B/n, multivariate) to measure the impact of product changes.
+- **[Product Analytics](/product-analytics/overview)**: Track key metrics that drive your product and business success.
+- **[Web Analytics](/webanalytics/overview)**: Understand how users interact with your web properties, track traffic, and analyze performance.
+- **[Session Replays](/session-replay/overview)**: Watch replays of user sessions to see exactly how users interact with your product and identify opportunities to improve.
+- **[AI/ML Optimization](/ai-ml/ai-prompt-experiment)**: Automatically optimize experiments and track the performance of AI prompts with Statsig’s AI capabilities.
+- **[Statsig Warehouse Native](/statsig-warehouse-native/introduction)**: Integrate directly with your data warehouse to run experiments on top of your existing datasets.
```

Comment:
I thought this was going to be a visual like https://latest.docs-v2.statsig.com/

that would tighten it up a bit and bring the getting started up higher

Reply:
agreed. The comments I have in the PR description includes this. Let me look into a way if I could quickly bring in the components from v2